### PR TITLE
v.0.73

### PR DESCRIPTION
**Pull request by lcamilo**

### CODE REVIEW

**Comment by lcamilo** [comment]
I tested some queries that have enough data and made the necessary corrections, others visually checked and corrected how I know they should work, but it will only be possible to test with certainty when I have more data in the database. If possible, you can test again with MySQL, it may be that some correction I made to work with MSSQL may stop working with MySQL.

**Comment by palinkas-jo-reggelt** [comment]
OK cool. I'll look at it tonight and test it out. If you want data, I can dump my db and put it here.

**Comment by lcamilo** [comment]
> OK cool. I'll look at it tonight and test it out. If you want data, I can dump my db and put it here.

Cool, it could be a good way. If you can export it as INSERTs, I can fix whatever is necessary to import it into my MSSQL database.

**Comment by palinkas-jo-reggelt** [comment]
Here you go. Zipped up because GitHub wouldn't take sql.


[hm_fwban.sql.zip](https://github.com/palinkas-jo-reggelt/hMailServer-Firewall-Ban/files/4188101/hm_fwban.sql.zip)

File: www/blocks-view.php [hunk 1]
```diff
@@ -25,6 +25,10 @@
 
 	$no_of_records_per_page = 20;
 	$offset = ($page-1) * $no_of_records_per_page;
+
+	#what this must be?
+	$search = "???";
+
```
Comment (palinkas-jo-reggelt) [inline www/blocks-view.php:31]: I don't understand the question. $offset is the query row offset.

There's no search feature on blocks-view.php. Just click the IP you want detail on.

Comment (lcamilo) [reply www/blocks-view.php:31]: > I don't understand the question. $offset is the query row offset.
> 
> There's no search feature on blocks-view.php. Just click the IP you want detail on.

The $ search variable was not declared / initialized and was used below in a message, it threw an error for me, so I "initialized" and marked this comment for you to check if this is correct or if the variable was being used in the message by mistake.

**Comment by palinkas-jo-reggelt** [comment]
[hm_fwban_rh.zip](https://github.com/palinkas-jo-reggelt/hMailServer-Firewall-Ban/files/4189503/hm_fwban_rh.zip)

Almost forgot - you need the blocks too. (hm_fwban_rh)

**Pull request merged**